Modules/_winapi.c

```diff
@@ -1699,6 +1699,80 @@ _winapi_GetFileType_impl(PyObject *module, HANDLE handle)
     return result;
 }
 
+/*[clinic input]
+_winapi.CreateJobObject -> HANDLE
+
+    limit_info: object
+    job_name: Py_UNICODE(accept={str, NoneType}) = NULL
+    /
+
+Creates and returns the handle for a new Job Object.
+
+[clinic start generated code]*/
+
+static HANDLE
+_winapi_CreateJobObject_impl(PyObject *module, PyObject *limit_info,
+                             Py_UNICODE *job_name)
+/*[clinic end generated code: output=ddb90fccf4f363b9 input=f1350a07abdab61c]*/
+{
+    HANDLE job_handle;
+    Py_BEGIN_ALLOW_THREADS
+    job_handle = CreateJobObjectW(NULL, job_name);
+    Py_END_ALLOW_THREADS
+
+    if (job_handle == NULL) {
+        PyErr_SetFromWindowsErr(GetLastError());
+        return NULL;
```

Comment:
`HANDLE_return_converter` requires `INVALID_HANDLE_VALUE` for an error.

Reply:
Good catch, thanks.